src/diffusers/pipeline_utils.py

```diff
@@ -784,3 +784,38 @@ def progress_bar(self, iterable=None, total=None):
 
     def set_progress_bar_config(self, **kwargs):
         self._progress_bar_config = kwargs
+
+    def enable_xformers_memory_efficient_attention(self):
```

Comment:
this enable and disable shorthands are just there because many derived pipelines were using that, so I figured that it was cheaper to expose the call here :)

Reply:
Makes sense to me to make a method of `DiffusionPipeline` !